frontend/components/CommandPalette/CommandPalette.tests.tsx

```
@@ -95,6 +110,38 @@ describe("CommandPalette", () => {
       expect(screen.getByPlaceholderText(/search/i)).toBeInTheDocument();
     });
 
+    it("Ctrl+K does NOT open the palette on macOS (Cmd is required)", async () => {
+      // Ctrl+K is readline kill-line on macOS — we must not hijack it.
+      const { user } = adminRender(<CommandPalette />);
+      await user.keyboard("{Control>}k{/Control}");
+
+      await new Promise((r) => setTimeout(r, 0));
+      expect(
+        screen.queryByPlaceholderText(/search/i)
+      ).not.toBeInTheDocument();
```

ESLint

[error] Replace `⏎········screen.queryByPlaceholderText(/search/i)⏎······` with `screen.queryByPlaceholderText(/search/i)`

(prettier/prettier)